helm-charts/support/values.jsonnet

```diff
@@ -127,6 +129,21 @@ local makeUserPodUnschedulableAlert = function(
   ],
 };
 
+local configCostMonitoring = {
+  enabled: true,
+  extraEnv: [
+    {
+      name: 'CLUSTER_NAME',
+      value: cluster_name,
+    },
+  ],
+  serviceAccount: {
+    annotations: {
+      'eks.amazonaws.com/role-arn': 'arn:aws:iam::%s:role/jupyterhub_cost_monitoring_iam_role' % aws_account_id,
```

Comment:
Add a comment with a link to the terraform line where we create this role? And in terraform, mention that if this name is changed, we must change this in jsonnet as well.

Reply:
https://github.com/2i2c-org/infrastructure/pull/6515/commits/eb19f4b041a014a64dfde506b3c830cf675ec464